LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceAPIService.cs

```diff
@@ -106,7 +107,7 @@ private async Task<SearchDevicesResult> SearchDevicesAsync(string gatewayID = nu
                 ["DevAddr"] = devAddr,
                 ["DevEUI"] = devEUI,
                 ["AppEUI"] = appEUI,
-                ["DevNonce"] = devNonce
+                ["DevNonce"] = devNonce?.ToString("N", CultureInfo.InvariantCulture)
```

Comment:
Does this need to be in hex? The nonce is just a number so it seems more natural just to express it in base 10 decimal.

Reply:
Reverted in 9f3eb36e738fbaa1a3f376ab154540aa659eb33c